### PR TITLE
Db connection

### PR DESCRIPTION
**Pull request by GabalisD**

### CODE REVIEW

File: dev-learning/Startup.cs [hunk 1]
```diff
@@ -16,12 +18,14 @@ public Startup(IConfiguration configuration)
...
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllers();
-            _ = services.AddDbContext<Models.UserContext>(opt =>
-                opt.UseInMemoryDatabase("Users"));
+            services.AddDbContextPool<UserContext>(
+               options => options.UseMySql(Configuration.GetConnectionString("dbConfig")
+            ));
+            services.AddMvc();
+
```
Comment (EdasL) [inline dev-learning/Startup.cs:28]: 28 line redundant empty line

File: dev-learning/Startup.cs [hunk 2]
```diff
@@ -16,12 +18,14 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
+
```
Comment (EdasL) [inline dev-learning/Startup.cs:21]: 21 line redundant empty line

File: dev-learning/Controllers/UsersController.cs [hunk 3]
```diff
@@ -68,14 +68,13 @@ public async Task<IActionResult> PutUser(int id, User user)
 
             return NoContent();
         }
-
+
```
Comment (EdasL) [inline dev-learning/Controllers/UsersController.cs]: Remove trailing whitespaces

File: dev-learning/Models/UserContext.cs [hunk 4]
```diff
@@ -12,11 +11,5 @@ public UserContext(DbContextOptions<UserContext> options)
...
         public DbSet<User> Users { get; set; }
 
-        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-        {
-            string connectionString = ConfigurationManager.ConnectionStrings["dbConfig"].ConnectionString;
-
-            _ = optionsBuilder.UseMySQL(connectionString);
-        }
     }
 }
```
Comment (EdasL) [inline dev-learning/Models/UserContext.cs]: Screenshot 2020-04-25 at 11 27 43

**Review by EdasL — APPROVED** [review]

**Review by NeriusZar — APPROVED** [review]

**Pull request merged**